.changeset/young-ravens-hammer.md

```diff
@@ -0,0 +1,5 @@
+---
+"@astrojs/netlify": patch
+---
+
+Fixes an issue where using `global.crypto` failed for older Astro versions
```

Comment:
Can we give a version number here to be more specific? Something like:

> failed for Astro vx.x and earlier

(Assuming this fix works for *all* earlier Astro versions. Otherwise, specifying the range would be a nice addition.)

Reply:
@lilnasy do you know which versions this applies to? I guess: `<v4.4.1`?

Reply:
Yes, it was fixed in 4.4.1.